backend-rust/src/graphql_api/baker_metrics.rs

```diff
@@ -70,14 +70,14 @@ impl QueryBakerMetrics {
             y_last_baker_count.push(current_period_baker_count);
         }
 
-        let last_baker_count = y_last_baker_count.last().copied().ok_or_else(|| {
+        let last_baker_count = y_last_baker_count.iter().max().ok_or_else(|| {
```

Comment:
If for example indexer is down we still have a total value

Reply:
Becauxe this is correct, the previous was not. I can also do it as part of another PR, but it has to be done like this.

Reply:
But if the count is 50 in the past and goes down to 40 in the latest block, this will return 50 instead of 40, right?

Reply:
Yes you are right. I was a thinking it was growing, but that is indeed a problem.

Ehh. That will probably be hard to fix for no occurring last bakers then. I should probably revert.